src/TodoList.js

```diff
@@ -2,28 +2,17 @@ import React from "react";
 import TodoListItem from "./TodoListItem";
 
 // function to list from todos from array
-function TodoList(){
-    const todoList = [
-        {
-          'id':1,
-          'title':'waitting assignment'
-        }, {
-          'id':2,
-          'title':'ongoing assignment'
-        }, {
-          'id':3,
-          'title':'Complete assignment'
-        },
-      ];
-    return (
+function TodoList(props){
+  console.log(props);
```

Comment:
Ensure that you are removing console.log statements after you are finished with them.